include/fastdds/rtps/writer/StatelessWriter.h

```diff
@@ -76,6 +76,12 @@ class StatelessWriter : public RTPSWriter
             WriterHistory* hist,
             WriterListener* listen = nullptr);
 
+    mutable LocatorList_t fixed_locators_;
+
+    virtual bool send_to_fixed_locators(
+            CDRMessage_t* message,
+            std::chrono::steady_clock::time_point& max_blocking_time_point) const;
```

Comment:
Moving an attribute and adding a virtual method aretechnically ABI breaks, but since the constructors of this class are protected, and not marked as exported, we can proceed with the proposed changes